gamefixes-steam/1144400.py

```diff
@@ -0,0 +1,13 @@
+""" Game fix for Senren Banka
+"""
+#pylint: disable=C0103
+
+from protonfixes import util
+
+def main():
+    """
+    """
+    util.protontricks('quartz')
+    util.protontricks('wmp9')
```

Comment:
Have you tested with `wmp11`?

Reply:
WMP11 install instead of WMP9 results in a black screen.